src/walletConnect/screens/SessionRequest.tsx

```diff
@@ -21,8 +23,24 @@ function SessionRequest({ pendingSession }: Props) {
 
   const phoneNumber = useSelector(e164NumberSelector)
   const address = useSelector(currentAccountSelector)
+  const walletConnectDisplayedInfo = useSelector(walletConnectDisplayedInfoSelector)
   const isDappListed = useIsDappListed(name)
 
+  const requestDetails =
+    walletConnectDisplayedInfo === WalletConnectDisplayedInfo.None
+      ? []
+      : [
+          {
+            label: t('phoneNumber'),
+            value: phoneNumber,
+          },
```

Comment:
Looks like we missed this in a previous PR, but phone number is never shared when WC is used. We should remove this from here.